components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -2280,35 +2333,35 @@ public AccessTokenDO retrieveLatestToken(Connection connection, String consumerK
             resultSet = prepStmt.executeQuery();
 
             if (resultSet.next()) {
-                    String accessToken = persistenceProcessor.getPreprocessedAccessTokenIdentifier(
-                            resultSet.getString(1));
-                    String refreshToken = null;
-                    if (resultSet.getString(2) != null) {
-                        refreshToken = persistenceProcessor.getPreprocessedRefreshToken(resultSet.getString(2));
-                    }
-                    long issuedTime = resultSet.getTimestamp(3, Calendar.getInstance(TimeZone.getTimeZone("UTC")))
-                            .getTime();
-                    long refreshTokenIssuedTime = resultSet.getTimestamp(4, Calendar.getInstance(TimeZone.getTimeZone
-                            ("UTC"))).getTime();
-                    long validityPeriodInMillis = resultSet.getLong(5);
-                    long refreshTokenValidityPeriodInMillis = resultSet.getLong(6);
-
-                    String userType = resultSet.getString(7);
-                    String tokenId = resultSet.getString(8);
-                    String subjectIdentifier = resultSet.getString(9);
-                    // data loss at dividing the validity period but can be neglected
-                    AuthenticatedUser user = new AuthenticatedUser();
-                    user.setUserName(tenantAwareUsernameWithNoUserDomain);
-                    user.setTenantDomain(tenantDomain);
-                    user.setUserStoreDomain(userDomain);
-                    user.setAuthenticatedSubjectIdentifier(subjectIdentifier);
-                    AccessTokenDO accessTokenDO = new AccessTokenDO(consumerKey, user, OAuth2Util.buildScopeArray
-                            (scope), new Timestamp(issuedTime), new Timestamp(refreshTokenIssuedTime)
-                            , validityPeriodInMillis, refreshTokenValidityPeriodInMillis, userType);
-                    accessTokenDO.setAccessToken(accessToken);
-                    accessTokenDO.setRefreshToken(refreshToken);
-                    accessTokenDO.setTokenId(tokenId);
-                    return accessTokenDO;
+                String accessToken = persistenceProcessor.getPreprocessedAccessTokenIdentifier(
+                        resultSet.getString(1));
+                String refreshToken = null;
+                if (resultSet.getString(2) != null) {
+                    refreshToken = persistenceProcessor.getPreprocessedRefreshToken(resultSet.getString(2));
+                }
+                long issuedTime = resultSet.getTimestamp(3, Calendar.getInstance(TimeZone.getTimeZone("UTC")))
```

Comment:
Use the UTC constant

Reply:
Fix with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472